core/services/chainlink/testdata/config-invalid.toml

```
@@ -202,6 +202,15 @@ Name = 'tron-test'
 [[Tron.Nodes]]
 Name = 'tron-test'
 
+[[Ton]]
+Enabled = 1
+
+[[Ton.Nodes]]
+Name = 'ton-test'
+
+[[Ton.Nodes]]
```

nit: we should capitalize TON across interface components, unless it's a all lowercase label/key `ton`:

```suggestion
[[TON]]
Enabled = 1

[[TON.Nodes]]
Name = 'ton-test'

[[TON.Nodes]]
```